### PR TITLE
[incubator/common] env from configmap

### PR DESCRIPTION
**Pull request by alebabai**
**What this PR does / why we need it**:
Add template to rendering env from configmap

### CODE REVIEW

**Comment by mattfarina** [comment]
@prydonius Can you please look at these changes?

File: incubator/common/templates/_envvar.tpl [hunk 1]
```diff
@@ -6,6 +6,18 @@
   value: {{ default "" $value | quote }}
 {{- end -}}
 
+{{- define "common.envvar.confimap" -}}
```
Comment (prydonius) [inline incubator/common/templates/_envvar.tpl]: typo: configmap

**Review by prydonius — COMMENTED** [review]
lgtm apart from the typo

File: incubator/common/templates/_envvar.tpl (hunk 1 above)
Comment (alebabai) [reply incubator/common/templates/_envvar.tpl]: fixed, thanks

**Comment by mattfarina** [comment]
/lgtm

Since the typo has been fixed and @prydonius was good with it

**Pull request merged**